app.js

```diff
@@ -92,14 +92,14 @@ io.sockets.on("connection", function (socket) {
         sink.on("exception", function (data) {
             socket.emit("exception", data);
         });
-        urlSafety.checkUrlSafety(data.url, function(err, res) {
-            if(!res) {
-                socket.emit("exception", {message: "error while resolving " + data.url + " Check the spelling of the host, the protocol (http, https) and ensure that the page is accessible from the public Internet."});
-            }
-            else {
+        insafe.check({
+            url: data.url,
+            statusCodesAccepted: ["301"]
+        }).then(function(res){
+            if(res.status == true) {
```

Comment:
In general, please use `===` instead of `==` to avoid type coercion.

But in that case, `if (res.status)` is enough.